src/NzbDrone.Core/Movies/Performers/RefreshPerformerService.cs

```diff
@@ -118,7 +122,48 @@ private void SyncPerformerItems(Performer performer)
                         },
                         Monitored = true,
                         Tags = performer.Tags
-                    }).ToList(), true);
+                    }).ToList()
+                        .Select((x, i) => new { Index = i, Value = x })
+                            .GroupBy(x => x.Index / chunkSize)
+                            .Select(x => x.Select(v => v.Value).ToList())
+                            .ToList();
```

Comment:
Using .Chunk(10) when filtering scenesToAdd @109 and looping the chunked list while adding will make for cleaner code.

Reply:
Now using Chunk()